docs/src/main/sphinx/release/release-382.md

```diff
@@ -0,0 +1,52 @@
+# Release 382 (25 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with [exchange spooling on Google Cloud Storage](fte-exchange-manager). ({issue}`12360`)
```

Comment:
@arhimondr @linzebing .. fyi .. both reworded a bit ..  I will work with @colebow and @jhlodin to get a follow up doc PR off the ground with some minor tweaks

Reply:
PR for that was merged. Thanks